lib/util.js

```diff
@@ -258,6 +259,7 @@ module.exports = {
   isFunction,
   isPrimitive,
   log,
+  MIME,
```

Comment:
I think the PR description and the docs are not matching the fact that mime is exposed via util.

Reply:
still rebasing, see last commit being `wip` and recent comments on the thread

Reply:
fixed